src/porepy/numerics/fv/mpfa.py

```diff
@@ -730,6 +730,7 @@ def _local_discr(
         sgn_arr[bound_faces] = g.cell_faces[bound_faces].sum(axis=1).A.ravel()
```

Comment:
Could you leave one line of comment why the sign corrections are necessary, and what they do?

Reply:
The honest answer is that I'm not entirely sure, but now it's consistent with the face contribution. Do you want me to write this as a comment?